CONTRIBUTING.md

```diff
@@ -34,6 +34,7 @@ In case you are suggesting a new feature, we will match your idea with our curre
 
 ### Setup with npm
 * Install the dependencies: `npm install`
+* Bootstrap the packages:`npx lerna bootstrap`
```

Comment:
No need to use npx, regular npm is sufficient

Reply:
You mean `npm lerna bootstrap`? It doesn't seem to work for me. I use npm@6.4.1

Reply:
Npm run bootstrap is the command you’re looking for I assume

Reply:
Yes, you're right. My initial thought was to keep consistent between yarn and npm.